lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -918,8 +916,13 @@ def install_puppetlabs_release_repo_on( hosts, repo = nil, opts = options )
                 variant_url_value = 'cisco-wrlinux'
                 version = '7'
               end
-              remote = "%s/puppetlabs-release%s-%s-%s.noarch.rpm" %
-                [opts[:release_yum_repo_url], repo_name, variant_url_value, version]
+              if repo_name == 'pc1'
+                remote = "%s/puppetlabs-release-%s-%s-%s.noarch.rpm" %
+                  [opts[:release_yum_repo_url], repo_name, variant_url_value, version]
+              else
+                remote = "%s/puppet%s/puppet%s-release-%s-%s.noarch.rpm" %
+                  [opts[:release_yum_repo_url], repo_name, repo_name, variant_url_value, version]
+              end
```

Comment:
@melissa so if I'm understanding this right, if people don't provide a `repo_name`, an example url beaker could generate is:
```
release_yum_repo_url/puppet/puppet-release-el-6.noarch.rpm
```
but to get puppet 5 or 6, they provide that as the `repo_name`, and the url example above would be:
```
release_yum_repo_url/puppet5/puppet5-release-el-6.noarch.arm
```
Is this correct?

Looking at how you've added in the backwards compatible case, it looks like the strings don't match up:
```
old: %s/puppetlabs-release%s-%s-%s.noarch.rpm
new: %s/puppetlabs-release-%s-%s-%s.noarch.rpm
```
Was the old one incorrect? Once we get that discrepancy settled, I'm down with leaving this as a strategy for backwards compatibility. If you would make a comment for us to try to make sure to come back & get rid of this case in the future, that'd be cool ([example](https://github.com/puppetlabs/beaker/blob/master/lib/beaker/host/pswindows/exec.rb#L183)).